thinkhazard/celery.py

```diff
@@ -32,11 +31,17 @@ def publish():
 @app.task
 def transifex_fetch():
     print("start transifex_fetch")
-    # FIXME plug to real task
-    time.sleep(10)
+    os.system("/app/thinkhazard/scripts/tx-pull-db")
```

Comment:
Does the output ends up the container log?

Reply:
I can find entries such as {"status": "SUCCESS", "result": null, "traceback": null, "children": [], "date_done": "2020-05-13T12:40:19.284408", "task_id": "3743b2b9-6a12-4a3d-98ec-53e856ef5feb"} on the s3 for the corresponding time.

Reply:
Done, note that os.sytem doc say to use subprocess methods instead.
I checked and output are done in realtime on stdout.
Note that I would prefer to remove S3 result backend completely and get everything on container stdout.

Reply:
Not sure if I understand right. Would you remove all celery logging to S3?

Regarding subprocess I was wondering which one to use. Also for my understanding, what's the advantage here?

Reply:
Note that logging is not not actually send to s3, only results, but we do not know about the command runned. I would prefer having the results at end of log.

Reply:
Concerning suprocess, the advantage here => we get the command output in container output.

Reply:
Ok I removed sending celery logs to S3 for now, which also resolves our problem between celery and minio in dev